Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# Makefile for lclusterd
+#
+# NOTE: the config.go present in this directory is copied to the client
+#       directory right before the `go build` step.
+#
+
+# 
+# phony targets
+#
+.PHONY: all client regen_proto clean 
+
+#
+# make targets
+#
+
+all: regen_proto lclusterd client 
+
+regen_proto:
+	@echo "Regenerating lclusterpb from prototype..."
+	@protoc -I lclusterpb/ lclusterpb/lclusterpb.proto --go_out=plugins=grpc:lclusterpb
+
+lclusterd:
+	@echo "Building lclusterd..."
+	@go build
+
+client:
+	@echo "Building lcluster client..."
+	@cp config.go client/config.go
```

Comment:
Done, I think turning it into the lcfg package is much cleaner.

https://github.com/rbisewski/lclusterd/pull/2/commits/e9314bcbff98426f69c8ae5ea753072a3c75611e

https://github.com/rbisewski/lclusterd/pull/2/commits/c4ebb6723a4dd142e896933c229321b19b2ce8ba